OWNERS_ALIASES

```diff
@@ -182,9 +182,11 @@ aliases:
   sig-docs-vi-owners: # Admins for Vietnamese content
     - ngtuna
     - truongnh1992
+    - huynguyennovem
```

Comment:
You have to alphabetize the name of owners

Reply:
Thanks! I have just fixed this.